portfolio/src/main/webapp/scripts/comments.js

```diff
@@ -15,6 +15,16 @@ async function fetchAndInsertDataIntoDOM() {
 	});
 }
 
+/*
+ * Deletes all comments from Datastore and 
+ * calls function to refresh comment page view
+ */
+function deleteAllCommentsFromStorage() {
+	fetch('delete-data', {
+		method: 'POST'
+	}).then(_ => this.fetchAndInsertDataIntoDOM())
```

Comment:
Interesting, I haven't seen the _ notation before. Is this using the underscore library to declare a function?

Reply:
No, it's not using the underscore library, it's just a notation that I picked up to use in promise chains when I don't need to use whatever resolved from the last thing in the chain. Not sure if it's common but I saw it all over the place during my last internship.

Reply:
Ah ok. I think typically we use this notation in our codebase for defining annonymous functions:

() => this.fetchAndInsertDataIntoDOM()

I might be misunderstanding what the underscore does though

Reply:
Interesting, I'll take to using that notation then from now on. The underscore doesn't actually do anything haha.